src/Proto.Actor/Mailbox/Mailbox.cs

```diff
@@ -10,7 +10,7 @@
 
 namespace Proto.Mailbox
 {
```

Comment:
What's the reason for changing these to public?

Reply:
Same for BoundedMailboxQueue

Reply:
We currently rely on the internalvisibleto thing for testing.
We could ofc still do that for the core lib. it just seems hacky.

Should we keep internal?

Reply:
I think internalvisibleto is ok as long as it won't create a lot of issues with signing

Reply:
For low-level unit tests I think InternalsVisibleTo is the way to go. Better than exposing internal classes in the API surface.

Reply:
(tbh in general I think we could and probably should make some other public classes internal)

Reply:
Brought back internal and also moved mailbox and router tests into same project as the actor tests.